examples/sagemaker/endpoint.yaml

```
@@ -0,0 +1,100 @@
+---
+apiVersion: sagemaker.aws.upbound.io/v1beta1
+kind: Endpoint
+metadata:
+  annotations:
+    meta.upbound.io/example-id: sagemaker/v1beta1/endpoint
+  labels:
+    testing.upbound.io/example-name: e
+    upjet.upbound.io/manual-intervention: "It requires an image which can be create/deleted only using AWS CLI"
+  name: e
+spec:
+  forProvider:
+    endpointConfigNameSelector:
+      matchLabels:
+        testing.upbound.io/example-name: ec
+    region: us-east-1
+    tags:
+      Name: foo
+---
+apiVersion: sagemaker.aws.upbound.io/v1beta1
+kind: EndpointConfiguration
+metadata:
+  annotations:
+    meta.upbound.io/example-id: sagemaker/v1beta1/endpointconfiguration
```

Could you please fix example-id?

```suggestion
    meta.upbound.io/example-id: sagemaker/v1beta1/endpoint
```